_el/exercises.md

```diff
@@ -1,169 +1,95 @@
 
-Write and run exercises can be combined with MCQs. For example, this MCQ
-can only be answered by running the Unix `ls` command:
+## Οι κλασσικές {#s: ασκήσεις-κλασικά}
 
-> **Combining MCQ with Code & Run**
-> 
-> You are in the directory `/home/greg`. Which of the following files is
-> *not* in that directory?
-> 
+Όπως συζητήθηκε στην Ενότητα [s:models-formative-assessment](#SECTION), οι *ερωτήσεις πολλαπλών επιλογών* (ΕΠΕ) είναι πιο αποτελεσματικές όταν οι λανθασμένες απαντήσεις βοηθούν στη διερεύνηση συγκεκριμένων παρανοήσεων. Σύμφωνα με την ταξινόμηση του Bloom ([s:process-objectives](#SECTION)), οι ΕΠΕ συνήθως σχεδιάζονται για να ελέγξουν την ανάκληση και κατανόηση ενός θέματος ("Ποια είναι η πρωτεύουσα του νομού Πέλλας;"), αλλά μπορούν επίσης να απαιτούν από τους εκπαιδευόμενους να ασκούν τις κριτικές τους ικανότητες.
+
+> ** Μια ερώτηση πολλαπλής επιλογής **
+>
+> Με ποιά σειρά γίνονται οι πράξεις όταν ο υπολογιστής αξιολογεί την έκφραση `price = addTaxes(cost - discount)`;
+>
+> 1. αφαίρεση, κλήση συνάρτησης, αντιστοίχιση
+> 1. κλήση συνάρτησης, αφαίρεση, αντιστοίχιση
+> 1. κλήση συνάρτησης, μετά αντιστοίχιση και αφαίρεση
+> 1. κανένα από τα παραπάνω
+
+Ένας δεύτερος κλασικός τύπος άσκησης προγραμματισμού είναι *κώδικας και εκτέλεση* (Κ&Ε), στην οποία ο εκπαιδευόμενος γράφει κώδικα ο οποίος παράγει ένα καθορισμένο αποτέλεσμα. Οι ασκήσεις Κ&Ε μπορούν να είναι από πολύ απλές έως αρκετά περίπλοκες ανάλογα με τις ανάγκες του διδασκάλου, αλλά για την ορθή χρήση τους εντός μιας αίθουσας διδασκαλίας, θα πρέπει να είναι σύντομη και να έχει μόνο μία ή δύο πιθανώς σωστές απαντήσεις. Σε ένα αρχάριο επίπεδο, αρκετά συχνά μπορεί να ζητήσετε από τους μαθητές να καλέσουν μια συγκεκριμένη συνάρτηση: οι έμπειροι εκπαιδευτικοί έχουν την τάση να ξεχνούν ότι μπορεί κάποιοι να βρίσκουν δύσκολη την έννοια της τοποθέτησης παραμέτρων στη σωστή σειρά. Για πιο προχωρημένους μαθητές, το να βρουν ποια συνάρτηση πρέπει να καλέσουν είναι συχνά πιο ελκυστικό και επομένως μια καλύτερη μετρική της κατανόησής τους.
```

Comment:
1. είναι *κώδικας --> είναι ο *κώδικας
2. στην οποία --> στον οποίο 
3. διδασκάλου--> δασκάλου
4. σύντομη και να έχει --> σύντομες και να έχουν
5. Σε ένα αρχάριο επίπεδο, αρκετά συχνά μπορεί να ζητήσετε από τους μαθητές --> Στους αρχάριους, είναι συχνά αρκετό να τους ζητήσετε
6. Για πιο προχωρημένους μαθητές --> Για τους πιο προχωρημένους μαθητές
7. μετρική--> μου ταιριάζει καλύτερα το "σημάδι" εδώ

Reply:
Thanks @pdelias ! Fixed in a new commit.